### PR TITLE
Split requests (#29)

### PR DESCRIPTION
**Pull request by sdelatorrep**
Each endpoint has its own Request object as requested in #29.

### CODE REVIEW

**Comment by mbaudis** [comment]
@sdelatorrep @jrambla Sorry, I don't get this. Wasn't the point in the discussion that this -s o far - does not make sense, since, obviously, one should be able to query all data components - individuals, variants, biosamples - for whatever path? Did I miss something - where? I thought the arguments (e.g. "query may specify conditions not matching e.g. the {id} in the path") were argued / solved?

**Comment by sdelatorrep** [comment]
I agree that we don't _really_ need this _now_, but @jrambla [requested](https://github.com/ga4gh-beacon/specification-v2/issues/29#issuecomment-807060900) me to do it:
> @sdelatorrep there is a way to use Dmitry's suggestion while keeping the request parameters common (like the filters object)?

**Comment by mbaudis** [comment]
@sdelatorrep I'm rather against this as a general multiplication of (mostly) identical schemas & would rather do changes in a "need based" way. It becomes really hard to parse this mentally, i.e. where schemas are same-same or where they need to be different.

In principle we have 2 general types of request: "info"-style requests and "bioobject" (variants, biosamples, individuals...) request (not analyzing here, just IMO). Maybe another type for variant evidences, annotations  but this probably is only different in the delivered schemas.  So we should rather think if this can be collapsed.

**Comment by sdelatorrep** [comment]
I agree with you, @mbaudis.
@jrambla , WDYT?

**Comment by jrambla** [comment]
We decided to revisit this, if necessary
